docs/design/properties_code_point_trie.md

```diff
@@ -0,0 +1,15 @@
+# Unicode Properties and Code Point Tries in ICU4X
+
+## Background
+
+[Unicode Properties](https://unicode-org.github.io/icu/userguide/strings/properties.html) represent attributes of code points in the Unicode specification. 
```

Comment:
There are a few different types of properties referred to in that link. I think it would be helpful to have a sentence explaining why we're considering just binary and enumerated properties in the rest of the document. Are they more important? Is it because they are suitable for storing in a Code Point Trie?